jf_agent/jf_jira/jira_download.py

```diff
@@ -33,9 +33,9 @@ def download_users(jira_connection, gdpr_active, quiet=False):
     jira_users = _search_all_users(jira_connection, gdpr_active)
 
     # Some jira instances won't return more than one page of
-    # results.  If we have seen exactly 1000 results, try
+    # results.  If we have seen approximately 1000 results, try
     # searching a different way
-    if len(jira_users) == 1000:
+    if 950 < len(jira_users) < 1000:
```

Comment:
This would no longer trigger on the old case (exactly 1000 results)... use `< 1001` or `<= 1000` instead?